unpub/lib/src/file_store.dart

```diff
@@ -1,13 +1,23 @@
 import 'dart:io';
+
 import 'package:path/path.dart' as path;
+
 import 'package_store.dart';
 
 class FileStore extends PackageStore {
   String baseDir;
+  bool asTree;
 
-  FileStore(this.baseDir);
+  FileStore(this.baseDir, {this.asTree: false});
```

Comment:
Would it be more general to replace this param with `filePath`?

Reply:
For example,

```dart
String Function(String name, String version)? getFilePath
```

Reply:
Sorry, not sure i follow. Given that its a dynamically generated path based upon the current package, I don't see how its user configurable?

Reply:
Yeah, it depends on the package name and version. We can pass a function to do it.

Reply:
ah, i think is see now what you are suggesting